kubernetes/assets/recommended_monitors/monitor_node_unavailable.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "[kubernetes] Monitor Unavailable Kubernetes Nodes",
+	"type": "query alert",
+	"query": "max(last_15m):sum:kubernetes_state.node.status{status:schedulable} by {kubernetes_cluster} * 100 / sum:kubernetes_state.node.status{*} by {kubernetes_cluster} < 80",
```

Comment:
notes:
not sure if people are still using 'schedulable' boolean.
this is useful, but would also be useful to monitor 'tainted' nodes.

Reply:
change this to measure not_ready instead of schedulable.